Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM golang:1.8
+RUN mkdir /code
+WORKDIR /code
+
+RUN go get github.com/golang/lint/golint
```

Comment:
Please use a package / dependency manager or at least install these deps from the Makefile